humility-core/src/core/gdb.rs

```diff
@@ -184,9 +193,9 @@ impl GDBCore {
 
     pub fn new(server: GDBServer) -> Result<GDBCore> {
         let port = match server {
-            GDBServer::OpenOCD => 3333,
-            GDBServer::JLink => 2331,
-            GDBServer::Qemu => 3333,
+            GDBServer::OpenOCD => OPENOCD_PORT,
```

Comment:
LGTM as-is but is it possible moving forwards that we'll have several concurrent instances of OpenOCD potentially on different ports?

Reply:
It should not be the case.  OpenOCD supports multiple targets through one openocd server.  Our current openocd core just connects to the default, but it should be possible to specify the correct target.  https://rivosinc.atlassian.net/browse/SW-1155